grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -339,6 +339,11 @@ class UrlMappings {
             action = [DELETE: "clearPicklist"]
         }
 
+        "/api/picklists/$id/items/$itemId" {
```

Comment:
This endpoint needs to be improved as it was discussed in the comment in service and on slack